qa-theme/SnowFlat/qa-styles-rtl.css

```diff
@@ -449,3 +449,11 @@ h1 {
 .qa-netvote-count-data {
 	direction: ltr;
 }
+
+[dir="auto"] code {
+    float: left;
+}
+
+[dir="rtl"] code {
```

Comment:
Why we've separated the `rtl` just for this theme? Is it needed? and if so, doesn't it apply to other themes as well?

Reply:
It was based on the body RTL attribute that we set on the rtl theme and is not needed for other themes ( because they don't have this property )